packages/integrations/lit/server.js

```diff
@@ -49,7 +49,7 @@ function* render(Component, attrs, slots) {
 
 	instance.connectedCallback();
 
-	yield `<${tagName}`;
+	yield `<${tagName} defer-hydration`;
```

Comment:
WDYM by non-serializable props? Do you mean props that are not reactive properties?

Reply:
I mean properties that don't serialize to attributes. If you use `reflect: true` you won't get a mismatch because it will use the attribute value. So in those cases it shouldn't need to wait and can hydrate immediately on define().

Reply:
For example, this test mostly is using `reflect: true`: https://github.com/withastro/astro/blob/main/packages/astro/test/fixtures/lit-element/src/components/my-element.ts#L13

Reply:
Oh, interesting. Does `client.js` run at a significantly later time after hydration trigger?

Reply:
You might use multiple hydration directives on the same page, one :load and one :visible. They should both hydrate on load but won't if they are deferred.

Reply:
makes sense. Done!